### DISCUSSION TITLE
Allow to set movie_max_size in configuration file

### DISCUSSION THREAD

**Discussion by angrycoding** [Motion-Project/motion#1486] [General]
Currently there is movie_max_time which is not good enough in case if backend has some limitations related with the file size itself (not it's duration). So it woulld be great if movie_max_size configuration option would allow me to set maximum size of the movie file in megabytes. 

; set maximum size of motion movies
movie_max_size = 50

My problem is that I'm sending those motion movies to the FTP server, which in my case has upload limitations, 50M per file. Currenly I've adjusted movie_max_time so final file is not overcoming this limit, but that is not very nice solution, cause same length movie can produce different sizes for different codes / resolutions / bps and so on.

**Comment by tosiara** [comment]
That may depend on codec. In case of CBR - that may be possible, but for VBR you cant be that precise. With extpipe this is not possible at all

**Comment by angrycoding** [comment]
I understand the limitations, but its way easier for motion to handle file size rather than for external app. Just would be great thing to have.
